Rock, paper, scissors/Model/Sign.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Sign.swift
+//  Rock, paper, scissors
+//
+//  Created by Константин Стародубцев on 13.07.2020.
+//  Copyright © 2020 Константин Стародубцев. All rights reserved.
+//
+
+import Foundation
+import GameplayKit
+
+// Data types for the "hand" signs and proper emoji
+// May be i should used Dictionary here instead enum...
+
+enum Sign: CaseIterable {
+    case rock, paper, scissors
+    var emoji: String {
+        switch self {
+        case .rock: return "👊🏼"
+        case .paper: return "✋🏼"
+        case .scissors: return "✌🏼"
+        }
+    }
+    var strongerThen: Sign {
+        switch self {
+        case .rock: return .scissors
+        case .paper: return .rock
+        case .scissors: return .paper
+        }
+    }
+}
+
+func getSignByEmoji(_ emoji: String) -> Sign {
+    for sign in Sign.allCases {
+        if sign.emoji == emoji {
+            return sign
+        }
+    }
+    preconditionFailure("Somewhy coudn't identify the sign by its emodji. May be emodji is wrong.")
+}
+
+let randomChoice = GKRandomDistribution(lowestValue: 0, highestValue: 2)
```

Comment:
Global variables and global functions are bad practices. Better move the var randomChoice and the function randomSign() to the Game class, because this logic is still a part of the game.